### PR TITLE
73/clipboard copy

### LINKED ISSUE #73: For form urls, add "Copy link" buttons

### PR DESCRIPTION
**Pull request by almet**
Add a button to copy the form URL to the clipboard. Fixes #73. Built on top of #187.

### CODE REVIEW

File: formbuilder/styles.css [hunk 1]
```diff
@@ -163,3 +164,15 @@ textarea.json-viewer {
 .background h1 {
   color: white !important;
 }
+
+.social {
```
Comment (rlecellier) [inline formbuilder/styles.css]: remove this, bootstrap class "list-inline" do it.

File: formbuilder/components/FormCreated.js [hunk 2]
```diff
@@ -1,28 +1,65 @@
...
+
+  Please, take some time to fill it,
+  `;
+
+    const emailUrl = `mailto:?subject=${emailSubject}&body=${encodeURIComponent(emailBody)}`;
+    return (
+      <form>
+        <h3>Neat, your form is now ready!</h3>
+        <div className="form-group">
+          <ul className="social">
```
Comment (rlecellier) [inline formbuilder/components/FormCreated.js]: use "list-inline"

File: formbuilder/components/URLDisplay.js [hunk 3]
```diff
@@ -6,12 +6,15 @@ export default function URLDisplay(props) {
...
 
   const icon = props.type === "admin" ? "eye-close" : "bullhorn";
+  const label = props.type == "admin" ? "Admin link" : "Form link";
   const glyphicon = `glyphicon glyphicon-${icon}`;
 
   return (
-    <div className="input-group input-group-lg">
-      <span className="input-group-addon"><i className={glyphicon} /></span>
-      <input onClick={onClick} type="text" className="form-control" defaultValue={props.url} />
+    <div>
```
Comment (rlecellier) [inline formbuilder/components/URLDisplay.js]: the display table isn't working here. i'll propose a fix

Comment (almet) [inline formbuilder/components/URLDisplay.js]: Can you tell me what's not working?

Comment (almet) [inline formbuilder/components/URLDisplay.js]: Ah, I think I got it, you're reffering to the fact that labels for the links aren't vertically aligned, correct?

Comment (rlecellier) [inline formbuilder/components/URLDisplay.js]: i made a mistake by looking at the version currently in http://kinto.github.io/formbuilder/
This is working.

**Pull request merged**